src/main/java/org/wiremock/grpc/dsl/WireMockGrpc.java

```diff
@@ -50,6 +52,17 @@ public static GrpcResponseDefinitionBuilder message(MessageOrBuilder messageOrBu
     return new GrpcResponseDefinitionBuilder(Status.OK).fromJson(json);
   }
 
+  public static GrpcResponseDefinitionBuilder messages(
```

Comment:
Suggest adding an overload taking a vararg so that it's not necessary to put `List.of(...)` when calling.